libs/cyrus-sasl/Makefile

```diff
@@ -8,14 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=cyrus-sasl
-PKG_VERSION:=2.1.26
-PKG_RELEASE:=3
+PKG_VERSION:=2.1.27
+PKG_RELEASE:=1
 
 PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=ftp://ftp.cyrusimap.org/cyrus-sasl/
-PKG_HASH:=8fbc5136512b59bb793657f36fadda6359cae3b08f01fd16b3d406f1345b7bc3
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-rc6.tar.gz
```

Comment:
Is it possibly work waiting for the official release an not a release candidate?

Reply:
This release candidate series has been going for quite some time. Nonetheless, we can wait.

Reply:
If you want to use the rc, you should adjust the PKG_VERSION to include "-rc6", so that you can avoid including the "-rc6" in PKG_SOURCE. And we would not be saying that the compiled package is already 2.1.27 (while it actually is 2.1.27-rc6).